src/wandbot/chat/config.py

```diff
@@ -21,18 +21,18 @@
 class ChatConfig(BaseSettings):
     chat_model_name: str = "gpt-4-1106-preview"
     max_retries: int = 2
-    fallback_model_name: str = "gpt-3.5-turbo-1106"
+    fallback_model_name: str = "gpt-4-1106-preview"
```

Comment:
@ayulockin : This fallback was supposed to be a model that can be used when GPT-4 calls fail. I know for evals this was set to 4, but we should use a fallback that is not the same model in prod.